src/main/scala/io/citrine/theta/benchmarks/StreamBenchmark.scala

```diff
@@ -7,7 +7,7 @@ import scala.util.Random
   * Created by maxhutch on 3/29/17.
   */
 class StreamBenchmark() extends Benchmark {
-  val N: Int = 33554432
+  val N: Int = 16777216
```

Comment:
why?

Reply:
The tests on travis were running out of heap space.  I first tried to increase the heap space via `<argLine>`, but the travis logs suggested that was getting ignored for some reason.  We don't want to be using that much heap anyways.